shared/chat/conversation-list/conversation-list-container.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import {namedConnect} from '../../util/container'
+import ConversationList from './conversation-list'
+
+type OwnProps = {
+  onSelect?: () => void,
+}
+
+const mapStateToProps = state => ({})
+const mapDispatchToProps = dispatch => ({})
+const mergeProps = (stateProps, dispatchProps, ownProps) => ({
+  rows: [],
```

Comment:
???

Reply:
Just a placeholder; Will be populated in a follow-up PRs